### DISCUSSION TITLE
401 access denied in downloading brunch r97

### DISCUSSION THREAD

**Discussion by zainlatif** [community/community#23551] [New to GitHub]
<p>i face problem while downloading “brunch r97 stable”<br>
access is denied 401 i also retype password and username but the popup is show continuously this application is not available on the another site so, solve this issue<br>
thanks!</p>

**Comment by jfullstackdev** [comment]
<p>can you give the exact link of that repo?  I suppose, it’s a repo on GitHub</p>

**Comment by zainlatif** [comment, marked as answer]
<p>Access denied problem is only with latest version now I install ‘brunch r88’ there was no problem</p>
